code/__HELPERS/string_num_lists.dm

```diff
@@ -0,0 +1,18 @@
+GLOBAL_LIST_EMPTY(string_num_lists)
```

Comment:
I'm unable to web-edit suggest all the changes needed:
* Turn this variable into `string_assoc_lists`
* Rename this file to `string_assoc_lists.dm`
* Replace `/string_num_list(` with `/string_assoc_list(` all around.
Then this PR will be ready for merge.

Reply:
yeah sorry i havnt been on my work pc, I saw it still needed changes but i didnt get to em yet, ill fix em  now.